ts/global.d.ts

```diff
@@ -35,6 +49,61 @@ declare function ParameterChanged(name: number, value: number): void;
 
 // ------ MIDI
 
+declare class MIDI {
+    /**
+     * Returns the MIDI note number for a given note name.
+     * For example: 'C3' or 'B#2'
+     * 
+     * Note: you cannot use flats in your argument. Use A#3, not Bb3
+     * 
+     * @param name - The name of the note.
+     */
+    public static noteNumber(name: string): number;
```

Comment:
I think `noteNumber` and `noteName` should use the `Pitch` enum instead of `number`.